### PR TITLE
feat: api create group, get my group

### PR DESCRIPTION
**Pull request by duchuyyyy**
# Pull Request types

- 🐛 Bug Fix
- 👍 Improvement
- ✨ New Feature
- 🎉 Release
- ♻️ Refactoring
- 🚿 Refactoring (Remove functions)
- 💚 Refactoring (Test or CI)
- 👕 Fix (Lint)
- 🆙 Update (Dependency packages)
- 🚀 Improvement (Performance)
- 👮 Improvement (Security)
- 🔒 Restriction (Restrict new feature)
- 👾 Others

# Description

# Checklist

- [ ] 1. Build successfully on local
- [ ] 2. Format all code files included in the PR
- [ ] 3. No more unconfirmed specs in the PR
- [ ] 4. Tested the normal case and error cases following specs
- [ ] 5. Is there a refactor to the old code? If yes, have you tested the old function and noted the impact yet?
- [ ] 6. Loops have a set length and correct termination conditions
- [ ] 7. There aren't any redundant or duplicate codes in PR (clean code)
- [ ] 8. There aren't any hard-coded in the code in PR
- [ ] 9. Optimization logic handling and optimizations SQL (for example, don't get redundant data or add an index to optimize performance, query only necessary fields...)
- [ ] 10. Update issue status and add PR link to the issue

# Evidence
(Screenshot or Video)

# Discussion

### CODE REVIEW

File: src/modules/group/repository/member-in-group.repository.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+  constructor(private readonly dataSource: DataSource) {
+    super(MemberInGroupEntity, dataSource.createEntityManager());
+  }
+
+  async getMyGroup(userId: Uuid, filterOptions: PageOptionsDto) {
+    const skip = filterOptions.page ? (filterOptions.page - 1) * filterOptions.limit : 0;
+
+    const query = this.createQueryBuilder('memberInGroup')
+      .leftJoinAndSelect('memberInGroup.group', 'group')
```
Comment (LeTatTuan) [inline src/modules/group/repository/member-in-group.repository.ts:18]: - nên dùng inner join cho đỡ data thừa khi join m

File: src/modules/group/member-in-group.service.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    private readonly memberInGroupRepository: MemberInGroupRepository,
+  ) {}
+
+  async addMemberToGroup(userId: Uuid, roleInGroup: RoleInGroup, group: GroupEntity) {
+    const memberInGroup = new MemberInGroupEntity({
+      memberId: userId,
+      groupId: group.id,
+      role: roleInGroup,
+    })
+    await this.memberInGroupRepository.save(memberInGroup);
```
Comment (LeTatTuan) [inline src/modules/group/member-in-group.service.ts:21]: - chưa có validate role host ms đc add vào m với user đã add vào thì có bắn lỗi ko 
- ko thì để host vào bảng group á

Comment (duchuyyyy) [reply src/modules/group/member-in-group.service.ts:21]: cái này đang làm viết hàm để dùng chung thôi m, còn validate thì validate ở func khác

Comment (LeTatTuan) [reply src/modules/group/member-in-group.service.ts:21]: ok m

**Pull request merged**